### ISSUE TITLE
Question on: rarefaction.single

### ISSUE THREAD

**Issue by itimling** [mothur/mothur.github.io#32]
Hello,
I want to rarefy an already existing OTU matrix (OTU_matrix.csv ).
According to the tutorial, I first need to create a  shared file. I downloaded the 'AmazonData' example data folder to run the example from the tutorial

mothur > make.shared(list=98_lt_phylip_amazon.fn.list, group=amazon.groups)

However, I get and error message
"No valid current files. You must provide a list or biom file before you can use the make.shared command.
[ERROR]: did not complete make.shared."

When I checked the data folder, I could not see the required file '98_lt_phylip_amazon.fn.list'

How can I proceed. with the example in order to see, how to adapt my existing OTU matrix?
Your help is greatly appreciated.
Ina

**Comment by pschloss** [comment]
Thanks for the heads up. Until we get this fixed, can you run the following from within mothur? It will get you the `98_lt_phylip_amazon.fn.list` file that you need to run the command...

```
cluster(phylip=98_lt_phylip_amazon.dist, name=amazon1.names, method=furthest, cutoff=0.03)
```

**Comment by itimling** [comment]
Hi Pat,
Thank you for the prompt response. 
Using the command you sent (cluster(phylip=98_lt_phylip_amazon.dist, name=amazon1.names, method=furthest, cutoff=0.03)) I got the file '98_lt_phylip_amazon.fn.list', which allowed me to run through the rarefaction tutorial. 
Nevertheless, I did not see an output that I was looking for, namely an OTU-matrix with rarefied OTU abundances that I could use for community analysis. How can I get such a rarefied OTU matrix?

Also, I do not understand how I can change me already existing OTU_matrix.csv file into an OTU_matrix.shared file. I tried adding the label, group and numOTUs as additional columns and then I gave it the extension .shared. However, the file was not recognized as such. 
I see in the tutorial (make.shared(list=98_lt_phylip_amazon.fn.list, group=amazon.groups), that in order to create a .shared file you use a list file with all the sequences from each sample that were assigned to an OTU. How can I make a .shared file from my already existing .csv file that has already OTU abundances?
Thank you very much for your help!!!
Ina
